components/ContributorCard.js

```diff
@@ -188,7 +188,7 @@ ContributorCard.propTypes = {
     description: PropTypes.string,
     collectiveSlug: PropTypes.string,
     isIncognito: PropTypes.bool,
-    type: PropTypes.oneOf(['USER', 'COLLECTIVE', 'ORGANIZATION', 'FUND', 'CHAPTER', 'ANONYMOUS']),
+    type: PropTypes.oneOf(['USER', 'COLLECTIVE', 'ORGANIZATION', 'EVENT', 'FUND', 'CHAPTER', 'ANONYMOUS']),
```

Comment:
Maybe you can read from constants CollectiveType instead.

Reply:
@znarf : Thanks. I've corrected that. Note that the `CollectiveType` doesn't contain, `CHAPTER` and `ANONYMOUS`; my understanding is that we don't use these anymore. Correct me if I am wrong here? 🤔